### PR TITLE
Tasks 968-969 : Xcopa

### PR DESCRIPTION
**Pull request by amirrezamirzaei**
creating two task from [Xcopa](https://github.com/cambridgeltl/xcopa)
they are the same tasks as in copa but in Estonian. this dataset also contains 10 more languages.  
 submitting this as a test to see whether we want these kinda task or not. each language contains 600 instances. should I make the other language tasks?

### CODE REVIEW

**Comment by danyaljj** [comment]
@Ravsehajsinghpuri wondering if you can review this PR?

File: tasks/task968_xcopa_commonsense_reasoning_et.json [hunk 1]
```diff
@@ -0,0 +1,3623 @@
...
+    ],
+    "Definition": "In this task you are given a premise and two alternatives in Estonian you must choose the alternative that is more plausibly the cause or effect of the situation described by the premise. The input format is \"premise\n(1)alternative_1(2)alternative_2\", the output should either be \"1\" or \"2\" based on your judgment.",
+    "Input_language": "et",
+    "Output_language": "en",
+    "Instruction_language": "en",
+    "Positive Examples": [
+        {
+            "input": "Ese oli mullikilesse mässitud.\n(1)See oli õrn.(2)See oli väike.",
+            "output": "1",
+            "explanation": "The first alternative makes more sense because you put object is bubble wrap if they are delicate."
```
Comment (Ravsehajsinghpuri) [inline tasks/task968_xcopa_commonsense_reasoning_et.json]: I think you meant "put object **in** bubble wrap"

File: tasks/task968_xcopa_commonsense_reasoning_et.json [hunk 2]
```diff
@@ -0,0 +1,3623 @@
...
+        },
+        {
+            "input": "Ma tühjendasin oma taskud.\n(1)Ma leidsin pileti tüki.(2)Ma leidsin relva.",
+            "output": "1",
+            "explanation": "The second alternative makes more sense because finding a ticket in your pocket is more common than finding a gun."
```
Comment (Ravsehajsinghpuri) [inline tasks/task968_xcopa_commonsense_reasoning_et.json]: The output is "1" but in the explanation you have mentioned "The second alternative makes more sense"

File: tasks/task968_xcopa_commonsense_reasoning_et.json [hunk 3]
```diff
@@ -0,0 +1,3623 @@
...
+        },
+        {
+            "input": "Termiidid tungisid majja sisse.\n(1)Termiidid kadusid majast.(2)Termiidid sõid läbi majas oleva puidu.",
+            "output": "2",
+            "explanation": "The second alternative makes more sense because the termites will eat wood."
+        }
+    ],
+    "Negative Examples": [
```
Comment (Ravsehajsinghpuri) [inline tasks/task968_xcopa_commonsense_reasoning_et.json:35]: Can you add one more negative example?

**Comment by Ravsehajsinghpuri** [comment]
> @Ravsehajsinghpuri wondering if you can review this PR?

sure!

**Comment by Ravsehajsinghpuri** [comment]
@amirrezamirzaei Can you update the "Input language" from et to Estonian and similarly for en to English for all the tasks? Similarly for "Output language" and "Instruction language". I think full form of the language is more appropriate.

**Comment by Ravsehajsinghpuri** [comment]
This PR is good to go when these minor changes are made. Rest all lgtm

**Comment by Ravsehajsinghpuri** [comment]
This PR lgtm @danyaljj

**Comment by danyaljj** [comment]
Please make sure to set the `"Categories": [""]` and `"Domains": [""]` fields according to [the taxonomy presented here](https://github.com/allenai/natural-instructions-expansion/blob/master/doc/task-hierarchy.md).

**Comment by amirrezamirzaei** [comment]
@danyaljj I'm not sure what the appropriate domain is in these two tasks. the instances don't seem to be about anything particular.

**Comment by Palipoor** [comment]
Hi! I think according to the paper, these are made from people's narratives, so I think "Personal Narratives" is a good domain. Please update the tasks in this pr and #386 accordingly. Thanks.

**Comment by swarooprm** [comment]
@amirrezamirzaei, can you update the files so that we can go ahead in this PR?

**Comment by aarunku5** [comment]
> @amirrezamirzaei, can you update the files so that we can go ahead in this PR?

@amirrezamirzaei reminder to address the reviewer comments please

**Comment by danyaljj** [comment]
Could you merge it with the main branch?

**Review by aarunku5 — APPROVED** [review]

**Pull request merged**